rollup.config.js

```diff
@@ -868,37 +868,46 @@ export default function rollup(options) {
   return builds;
 }
 
+async function triggerLiveReload(appDir) {
+  // Tickle live reload by touching the server entry.  Include all extensions
+  // since they may or may not use TS and/or JSX
```

Comment:
```suggestion
  // Tickle live reload by touching the server entry
  // Consider all of entry.server.{tsx,ts,jsx,js} since React may be used 
  // via `React.createElement` without the need for JSX.
```